### ISSUE TITLE
Add to functions template repo

### ISSUE THREAD

**Issue by TylerLeonhardt** [Azure/azure-functions-powershell-worker#44]
Similar to C# and JavaScript, we want to provide function templates in Powershell for commonly used trigger scenarios - 

- [x] Http Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/HttpTrigger-JavaScript))

- [x] Timer Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/TimerTrigger-JavaScript))

- [x] Blob Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/BlobTrigger-JavaScript))

- [x] Queue Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/QueueTrigger-JavaScript))

- [x] Event Grid Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/EventGridTrigger-JavaScript))

- [x] Cosmos DB Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/CosmosDBTrigger-JavaScript))

- [x] Event Hub Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/EventHubTrigger-JavaScript))

- [x] Service Bus Queue Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/ServiceBusQueueTrigger-JavaScript))

- [x] Service Bus Topic Trigger ([JS example](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/ServiceBusTopicTrigger-JavaScript))

We should have `HttpTrigger` and `TimerTrigger` templates for the private preview. See [this comment](https://github.com/Azure/azure-functions-powershell-worker/issues/44#issuecomment-426726355).

**Comment by asavaritayal** [comment]
Tweaked the description a little bit:) The right place for the templates is this repo - https://github.com/Azure/azure-functions-templates . @soninaren will be able to help with any questions!

**Comment by tylerl0706** [comment]
I believe @daxian-dbw already has a PR out for this somewhere. So I'll let him respond. I like the new description! 👍

**Comment by daxian-dbw** [comment]
The template PR is out there for a while: https://github.com/Azure/azure-functions-templates/pull/736

**Comment by soninaren** [comment]
@daxian-dbw, Had forgotten to tag templates PR as blocked. It was left as is because we had to wait for announcements related to PowerShell at Ignite. I should be able to review and merge the templates PR.

**Comment by daxian-dbw** [comment]
Thanks @soninaren. The template PR currently only contains the HTTP trigger template. We will submit PRs for templates of other triggers when we get to them.

**Comment by tylerl0706** [comment]
Spoke to @asavaritayal offline about this. We should have HttpTrigger and TimerTrigger templates for Private Preview

**Comment by tylerl0706** [comment]
Timer, Blob, Queue trigger coming soon - https://github.com/Azure/azure-functions-templates/pull/794

Cosmos DB trigger verified but found a bug in the worker. fixed in #83.

**Comment by joeyaiello** [comment]
Prototype only blocked on timer and HTTP, we can remove the milestone once timer is merged into the other repo.

**Comment by tylerl0706** [comment]
Timer, blob, queue have been merged 😊

**Comment by tylerl0706** [comment]
Just want to verify that Timer has been released before moving this out of Prototype

**Comment by TylerLeonhardt** [comment]
This is now complete and you can see the templates show up in core tools.
